answerking_app/tests/test_orders.py

```diff
@@ -460,9 +480,13 @@ def test_put_valid_status_returns_ok(self):
 
     def test_put_invalid_address_returns_bad_request(self):
         # Arrange
-        invalid_post_data: NewOrderAddressType = {"address": "test&"}
-        expected_failure_error: dict[str, list[str]] = {
-            "address": ["Enter a valid value."]
+        invalid_post_data: OrderType = {"address": "test&"}
+        expected: DetailError = {
+            "detail": "Validation Error",
+            "errors": {"address": ["Enter a valid value."]},
```

Comment:
can we highlight the characters that are not allowed

Reply:
?